Sprintfinity3902/Dungeon/Dungeon.cs

```diff
@@ -18,73 +28,85 @@ public class Dungeon : IDungeon
 
         private Game1 Game;
         private List<IRoom> dungeonRooms;
+        private IEntity boomerangItem;
+        private IEntity bombItem;
+        private IEntity movingSword;
+        private IEntity bowArrow;
+        private IEntity bombExplosion;
+        private IEntity hitboxSword;
         public IRoom CurrentRoom { get; set; }
-        private int currentId;
-        public int NextId { get; set; }
 
         private string backgroundMusicInstanceID;
 
+
+        public List<IEntity> linkProj;
+
         public Dungeon(Game1 game)
         {
+
+            boomerangItem = new BoomerangItem();
+            bombExplosion = new BombExplosionItem(new Vector2(-1000, -1000));
+            bombItem = new BombItem(new Vector2(-1000, -1000), (BombExplosionItem)bombExplosion);
+            movingSword = new MovingSwordItem(new Vector2(-1000, -1000));
+            hitboxSword = new SwordHitboxItem(new Vector2(-1000, -1000));
+            bowArrow = new ArrowItem(new Vector2(-1000, -1000));
+
             dungeonRooms = new List<IRoom>();
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room1.csv", 1));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room2.csv", 2));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room3.csv", 3));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room4.csv", 4));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room5.csv", 5));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room6.csv", 6));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room7.csv", 7));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room8.csv", 8));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room9.csv", 9));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room10.csv", 10));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room11.csv", 11));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room12.csv", 12));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room13.csv", 13));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room14.csv", 14));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room15.csv", 15));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room16.csv", 16));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room17.csv", 17));
-            dungeonRooms.Add(new Room(@"..\..\..\Content\Rooms\Room18.csv", 18));
 
-            CurrentRoom = GetById(1);
-            currentId = CurrentRoom.Id;
-            NextId = CurrentRoom.Id;
+            for (int roomNum = 1; roomNum < 19; roomNum++) {
```

Comment:
I love this haha!

Reply:
@Bailey-Ardrey Dude this is a meme haha!